opencensus/stats/internal/stats_exporter.cc

```diff
@@ -90,31 +96,51 @@ void StatsExporterImpl::StartExportThread() EXCLUSIVE_LOCKS_REQUIRED(mu_) {
 }
 
 void StatsExporterImpl::RunWorkerLoop() {
-  absl::Time next_export_time = absl::Now() + export_interval_;
+  absl::Time next_export_time;
+  {
+    absl::MutexLock l(&mu_);
+    next_export_time = absl::Now() + export_interval_;
```

Comment:
Make this block a function? To avoid repeating?

Reply:
Done.